docs/docsource/reference/blobserver/newlinks-final

```diff
@@ -0,0 +1,11 @@
+.. |hot-cool-archive| replace:: `Azure Blob storage: hot, cool, and archive access tiers <https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers>`__
```

Comment:
extra file?

Reply:
tidied up all my cruft (in this directory, at least!) -- sorry it got committed...